packages/login/src/components/TwoStepLoginForm/TwoStepLoginForm.js

```diff
@@ -0,0 +1,76 @@
+import React, {Component} from 'react'
+import {FormattedMessage, intlShape} from 'react-intl'
+import * as Tocco from 'tocco-ui'
+
+import '../Login/styles.scss'
+
+export class TwoStepLoginForm extends Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      userCode: ''
+    }
+  }
+
+  handleSubmit(e) {
+    e.preventDefault()
+    this.props.twoStepLogin(this.props.username, this.props.password, this.props.requestedCode, this.state.userCode)
+  }
+
+  handleUserCodeChange(e) {
+    this.setState({
+      userCode: e.target.value
+    })
+  }
+
+  render() {
+    return (
+      <div className="login-form">
+        {this.props.showTitle && <h1><FormattedMessage id="client.login.form.title"/></h1>}
+        <form>
```

Comment:
I think we should register the `handleSubmit` event handler on the form (not as click handler on the button). And the submit button should be of type `submit` (without click handler).